willa/tind/fetch.py

```diff
@@ -62,3 +68,86 @@ def fetch_file(file_url: str, output_dir: str = '') -> str:
         raise RecordNotFoundError('Referenced file could not be downloaded.')
 
     return saved_to
+
+def fetch_file_metadata(record: str) -> Record:
+    """Fetch file metadata for a given Tind record.
+    :raises AuthorizationError: When the TIND API key is invalid.
+    :raises Exception: for any response other then 200. 
+    :returns a json list of metadata for a given Tind record
+    """ 
+
+    status, files = tind_get(f"record/{record}/files")
+
+    if status != 200 and status < 500:
```

Comment:
Statuses >= 500 are already handled in `tind_get`.

```suggestion
    if status != 200:
```